### PR TITLE
Sign CLA as Coop IT Easy

### PR DESCRIPTION
**Pull request by carmenbianca**
Signed-off-by: Carmen Bianca Bakker <carmen@coopiteasy.be>

### CODE REVIEW

**Review by robinkeunen — CHANGES_REQUESTED** [review]

File: doc/cla/corporate/coopiteasy.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+Catherine Lembrée <catherine@coopiteasy.be> https://github.com/cathLemb
+
+Hugues De Keyzer <odoo@hugues.info> https://github.com/huguesdk
+
+Pol Champion <pol@coopiteasy.be> https://github.com/polchampion
+
+Rémy Taymans <remy@coopiteasy.be> https://github.com/remytms
+
+Robin Keunen <robin@coopiteasy.be> https://github.com/robinkeunen
```
[the remark this answers is not in the document]
Comment (carmenbianca) [reply doc/cla/corporate/coopiteasy.md:25]: changed

**Review by robinkeunen — APPROVED** [review]

File: doc/cla/corporate/coopiteasy.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Carmen Bianca Bakker <carmen@coopiteasy.be> https://github.com/carmenbianca
+
+Catherine Lembrée <catherine@coopiteasy.be> https://github.com/cathLemb
+
+Hugues De Keyzer <odoo@hugues.info> https://github.com/huguesdk
+
+Pol Champion <pol@coopiteasy.be> https://github.com/polchampion
+
+Rémy Taymans <remy@coopiteasy.be> https://github.com/remytms
```
Comment (remytms) [inline doc/cla/corporate/coopiteasy.md:23]: I use remytms AT tsmail.eu in my commits.

**Review by remytms — CHANGES_REQUESTED** [review]

File: doc/cla/corporate/coopiteasy.md (hunk 2 above)
Comment (carmenbianca) [reply doc/cla/corporate/coopiteasy.md]: done

**Review by remytms — APPROVED** [review]
Ok for me.

**Comment by mart-e** [comment]
Hi !

Thanks for the signature ! Could you just change your commit message to something like `[CLA] signature for Coop IT Easy` ?
Thanks

**Comment by carmenbianca** [comment]
@mart-e Done :+1:

**Comment by mart-e** [comment]
Thanks, also I just saw you made the PR against the master branch. This is fine but if you wish to make a contribution on a lower version, you can retarget on the 14.0 and we will forward port it on upper versions. If you it's not planned, you can keep it here.

**Comment by carmenbianca** [comment]
@mart-e Also done! :+1:

**Comment by mart-e** [comment]
Great thanks !
@robodoo r+

**Pull request closed without merging**